nx-X11/programs/Xserver/hw/nxagent/man/nxagent.1

```diff
@@ -497,6 +497,10 @@ XKEYBOARD will most likely not match the experienced settings. This is
 required on MacOS X to handle some keyboard problems that are special
 for this platform.
 .TP 8
+.I clone
+ask the real X server for the keyboard settings using XKEYBOARD protocol
+functions and clone them. This is the recommend setting.
```

Comment:
also in the commit msg of the "add keyboard=clone configuration option" commit:

s/Should the default but is/Should be the default but is/